src/main/java/com/autonomic/tmc/auth/exception/AuthSdkServiceException.java

```diff
@@ -19,23 +19,23 @@
  */
 package com.autonomic.tmc.auth.exception;
 
-import static com.autonomic.tmc.auth.exception.ErrorSourceType.SERVICE;
 import static java.util.Optional.ofNullable;
 
 import com.nimbusds.oauth2.sdk.ErrorObject;
 import com.nimbusds.oauth2.sdk.TokenErrorResponse;
 import java.util.Objects;
 
-public class SdkServiceException extends BaseSdkException {
+@SuppressWarnings("squid:S110")
+public class AuthSdkServiceException extends com.autonomic.tmc.exception.SdkServiceException {
 
     private int httpStatusCode = 500;
```

Comment:
IntelliJ is telling me that `getHttpStatusCode` method is unused. 

Can we write a test that sets the http error code and then validates using this getter?

Reply:
The method `getHttpStatusCode` is unused, but the property `httpStatusCode` value is tested in the `ClientCredentialsTokenSupplierTest` class. So we removed the method and are using a `@Getter` now.